src/Steps/Attachment1Graphics.tsx

```diff
@@ -45,41 +48,55 @@ export const Attachment1Graphics = () => {
         "DWG files shall not be a mix of vector and raster images. Vector only",
         "Any source data used in aiding of creation of DWG (templates, etc.) will be deleted from illustration prior to saving/exporting",
         "If multiple layers are used, then image is to be flattened into 1 single layer before exporting to minimize file size and reduce possible printing issues"
-      ]
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_dwg || false)
     },
     {
       Title: "Joint Photographic Expert Group (JPEG)",
       ID: "graphics_jpg",
       Descriptions: [
         "JPEG File format should be used as last resort.  Color screenshots/photos should be saved as TIFFs due to possible system compatibility issues",
         "B&W Bitmaps should not be saved as JPEGs"
-      ]
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_jpg || false)
     },
     {
       Title: "Portable Network Graphics (PNG)",
       ID: "graphics_png",
       Descriptions: [
         "Used for raster graphics",
         "Supports lossless data compression"
-      ]
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_png || false)
     },
     {
       Title: "Bitmap (BMP)",
       ID: "graphics_bmp",
       Descriptions: [
         "Used for raster graphics"
-      ]
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_bmp || false)
     },
     {
       Title: "Scalable Vector Graphics (SVG)",
       ID: "graphics_svg",
       Descriptions: [
         "Two-dimensional vector and mixed vector/raster graphics"
-      ]
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_svg || false)
+    },
+    {
+      Title: "No Graphics",
+      ID: "graphics_none",
+      Descriptions: [
+        "No graphics files required"
+      ],
+      Checked: (globalState.wizardOptions[globalState.tmcrIndex].graphics_none || false)
     }
-
   ];
 
+  const isChecked = guidelines.some( item => item.Checked === true);
```

Comment:
@RobertPorterfield -- See you noted this as a known issue.  Assume you are tracking these somewhere?